### ISSUE TITLE
Digits-DG data list

### ISSUE THREAD

**Issue by ZMC-IIIM** [zzwdx1234/MEDIC#3]

**Comment by zzwdx** [comment]
We download Digits-DG from this [link](https://drive.google.com/uc?id=15V7EsHfCcfbKgsDmzQKj_DfXt_XYp_P7&export=download). The data list is not required because the training set and validation set have already been divided in this dataset.

**Comment by ZMC-IIIM** [comment]
Thank you for your previous response. However, I have encountered some issues while training on the Digit-DG dataset, which have resulted in difficulty achieving higher accuracy. Additionally, adjusting the learning rate and batch size has not led to significant improvements. Specifically, while keeping the network architecture and parameters unchanged, I obtained an accuracy of 75.85% on the syn closed-set, 95.77% on minist, and 63.73% on minist_m. I am looking forward to your response. Here are some examples of training parameters.   Loading path...
Save name: syn
Save best test: False
Save later: False
Loading dataset...
Group length: 1
DataSet: DigitsDG
Source domain: ['mnist', 'mnist_m', 'svhn']
Target domain: ['syn']
Known classes: ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9']
Batch size: 8
CrossVal: True
Loading models...
Network: convnet
Number of epoch: 10000
Learning rate: 0.0001
Meta learning rate: 0.01
Share Parameter: False
Start training...
